Unable to use status-go fork to build status-react

Currently when a fork of `status-go` is used that isn't from `status-go` org it fails to build due to Go requirements for module paths:
```
../../go/src/github.com/jakubgs/status-go/mobile/response.go:6:2: cannot find package "github.com/status-im/status-go/account" in any of:
        /build/go/src/github.com/jakubgs/status-go/vendor/github.com/status-im/status-go/account (vendor tree)
        /nix/store/7xnvpkf3231kz2s1kx30c17v605cxz5s-go-1.14.7/share/go/src/github.com/status-im/status-go/account (from $GOROOT)
        /build/gomobile-work/src/github.com/status-im/status-go/account (from $GOPATH)
        /nix/store/33gdlax4x9125ndqx0vb05371qdhk14q-gomobile-20200622-33b8054/src/github.com/status-im/status-go/account
        /build/go/src/github.com/status-im/status-go/account
../../go/src/github.com/jakubgs/status-go/mobile/status.go:15:2: cannot find package "github.com/status-im/status-go/api" in any of:
```

--- a/nix/status-go/source.nix
+++ b/nix/status-go/source.nix
@@ -44,7 +44,8 @@ let
     shortRev = strings.substring 0 7 rev;
     rawVersion = versionJSON.version;
     cleanVersion = utils.sanitizeVersion versionJSON.version;
-    goPackagePath = "github.com/${owner}/${repo}";
+    # Need to pretend this is from status-im to let Go build it.
+    goPackagePath = "github.com/status-im/${repo}";
     src = fetchFromGitHub {
       inherit rev owner repo sha256;
       name = "${repo}-${shortRev}-source";

